dispatch/api/validators.py

```diff
@@ -60,3 +60,20 @@ def AuthorValidator(data):
         if 'type' in author and not isinstance(author['type'], basestring):
             # If type is defined, it should be a string
             raise ValidationError('The author type must be a string.')
+
+class ColumnSlugValidator(object):
```

Comment:
Instead of making a separate slug validator, I think you can just use `SlugValidator`. IMO it doesn't matter if a column and a section share the same slug.

Reply:
So here's where I ran into some confusion with how we structure the urls. I was deciding between whether I should do `baseurl/section_slug/column_slug/` or `baseurl/column_slug/` . I ended up choosing `baseurl/column_slug` because I thought it would be easier to enforce columns and sections not having the same slugs over columns and articles not having the same slug